presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotConfig.java

```diff
@@ -38,6 +38,7 @@
     public static final int DEFAULT_MIN_CONNECTIONS_PER_SERVER = 10;
     public static final int DEFAULT_THREAD_POOL_SIZE = 30;
     public static final int DEFAULT_NON_AGGREGATE_LIMIT_FOR_BROKER_QUERIES = 25_000;
+    public static final int DEFAULT_STREAMING_SERVER_GRPC_MAX_INBOUND_MESSAGE_SIZE = 128 * 1024 * 1024;
```

Comment:
- `_SIZE` -> `_BYTES`
- Either `128 * 1024 * 1024` -> `new DataSize(128, MEGABYTE).toBytes()` or just make the type to be `DataSize`. We support `DataSize` from config